README.md

```diff
@@ -123,6 +123,8 @@ Will send a log to Logz.io that looks like this:
 ```
 
 ### Release notes
+ - 1.0.18
+   - added `compressRequests` parameter to enable gzip compression of the logs before they are sent.
```

Comment:
+ allow to use system properties (if env variable does not exist ) in additionalFields , logzioUrl and token.
I think you should add it to "Parameters" section too.

Reply:
added option to inject system property value into additionalFields, logzioUrl and token.